vxmessenger/transport.py

```diff
@@ -175,6 +183,16 @@ class MessengerTransport(HttpRpcTransport):
     def setup_transport(self):
         yield super(MessengerTransport, self).setup_transport()
         self.pool = HTTPConnectionPool(self.clock, persistent=False)
+        self.BATCH_API_URL = self.config.get('outbound_url',
+                                             'https://graph.facebook.com')
+
+        static_config = self.get_static_config()
+        self.batch_size = static_config.request_batch_size
+        self.batch_time = static_config.request_batch_wait_time
+
+        self.redis = yield TxRedisManager.from_config(
+            static_config.redis_manager)
```

Comment:
redis_manager is not required, so what happens when I don't provide it? What happens when redis is not available?
Consider disabling batching if redis isn't set up properly?